src/resources/views/about/partials/donate.blade.php

```diff
@@ -0,0 +1,24 @@
+<div class="card card-solid">
+
+    <div class="card-header with-border">
+        <h3 class="card-title">
+            <i class="fas fa-donate"></i>
+            Donations
+        </h3>
+    </div>
+    <div class="card-body">
+        <div class="media">
+            <img class="mr-3" src="https://images.evetech.net/corporations/98482334/logo?size=128" alt="eveseat.net" width="64" height="64" />
+            <div class="media-body">
+                <p class="text-justify">
+                    If you like <a href="http://github.com/eveseat/seat/" target="_blank">SeAT</a> and want to tips the community behind amazing shipped features, you can send ISKs to <b>eveseat.net</b> corporation.
```

Comment:
> If you like <a href="http://github.com/eveseat/seat/" target="_blank">SeAT</a> and want to tips the community behind amazing shipped features, you can send ISKs to <b>eveseat.net</b> corporation.

If you like SeAT and want to tip the community behind it, you can send ISK to the <b>eveseat.net</b> corporation.

> This holding is used as a giant wallet and split his balance monthly across all contributors !

This holding corporation is used to split donations monthly across all contributors.